lib/pbench/test/unit/server/database/test_index_map_db.py

```diff
@@ -110,19 +114,24 @@ def test_merge_none(self, db_session, attach_dataset, to):
         drb = Dataset.query(name="drb")
         assert not IndexMap.exists(drb)
 
-        if to:
+        if orig:
             IndexMap.create(drb, map)
-            IndexMap.merge(drb, {})
-        else:
+
+        if merge:
             IndexMap.merge(drb, map)
+        else:
+            IndexMap.merge(drb, {})
```

Comment:
Obvious this this fine, but I would have used a single call and a ternary:
```python
        IndexMap.merge(drb, map if merge else {})
```
and, I might have been inclined to do something similar for the `orig`/`create` call.